charts/fybrik/files/adminconfig/default_policies.rego

```diff
@@ -4,35 +4,35 @@ package adminconfig
 
 # read capability deployment
 config[{"capability": "read", "decision": decision}] {
-    input.request.usage.read == true
+    input.request.usage[_] == "read"
```

Comment:
Why are policies changed if this is a single flow?

Reply:
Leaving it as a map makes usage more complex to populate, especially when we want to allow custom flow definitions

Reply:
OK.  Will require updates to some of the tests in rego_file_evaluator_test.go in a follow up PR.